tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed

```diff
@@ -974,3 +974,13 @@ class Test3 {
   }
 
 }
+
+/**
+ * Test that docblock is optoinal for __construct methods.
```

Comment:
nit: optional

Reply:
Thank you. Fixed. It would be nice to have `cspell` running here. I don't know if that is available for contrib, or how to set it up. I will investigate.